olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -75,146 +74,167 @@ class VersionedLayerClientImpl;
  * client_settings.task_scheduler = std::move(task_scheduler);
  * client_settings.network_request_handler = std::move(http_client);
  *
- * VersionedLayerClient client{"hrn:here:data:::your-catalog-hrn" "your-layer-id", client_settings};
- * auto callback = [](olp::client::ApiResponse<olp::model::Data, olp::client::ApiError> response) {};
- * auto request = DataRequest().WithVersion(100).WithPartitionId("269");
- * auto token = client.GetData(request, callback);
+ * VersionedLayerClient client{"hrn:here:data:::your-catalog-hrn"
+ * "your-layer-id", client_settings}; auto callback =
+ * [](olp::client::ApiResponse<olp::model::Data, olp::client::ApiError>
+ * response) {}; auto request =
+ * DataRequest().WithVersion(100).WithPartitionId("269"); auto token =
+ * client.GetData(request, callback);
```

Comment:
I'd try to keep this block away from the auto-format. As it became pretty much unreadable.
To mark a section as skipped, you can use `// clang-format off` comment. For example:
```cpp
// clang-format off
// Don't touch this, Clang!
connect( m_Job, SIGNAL( error( const QString&, const QString& ) ),  this, SLOT( onError( const QString&, const QString& ) ) );
connect( m_Job, SIGNAL( message( const QString& ) ),                this, SLOT( onMessage( const QString& ) ) );
connect( m_Job, SIGNAL( progress( int, int ) ),                     this, SLOT( onProgress( int, int ) ) );
// clang-format on
// Carry on formatting
```

Reply:
Done